.github/workflows/weekly-pulumi-update.yml

```diff
@@ -35,22 +35,33 @@ jobs:
       id: gomod
       run: >-
         git config --local user.email 'bot@pulumi.com'
+
         git config --local user.name 'pulumi-bot'
+
         git checkout -b update-pulumi/${{ github.run_id }}-${{ github.run_number }}
+
         go get github.com/pulumi/pulumi/pkg/v3
+
         go get github.com/pulumi/pulumi/sdk/v3
+
         go mod download
+
         go mod tidy
+
         git update-index -q --refresh
+
         if ! git diff-files --quiet; then
           echo ::set-output name=changes::1
         fi
-    - name: Provider with Pulumi Upgrade
+    - name: Bridge Pulumi Upgrade
       if: steps.gomod.outputs.changes != 0
       run: >-
         make build && make lint && make test
+
         git add .
+
         git commit -m "Updated modules"
+
```

Comment:
This is a cool workflow, TIL about repo-sync/pull-request@v2.6.2 automation piece! Nice!